code/src/java/pcgen/system/BatchExporter.java

```diff
@@ -390,13 +381,7 @@ public static boolean exportPartyToNonPDF(PartyFacade party, File outFile, File
 		{
 			party.export(new ExportHandler(templateFile), bw);
 			return true;
-		}
-		catch (final UnsupportedEncodingException e)
-		{
-			Logging.errorPrint("Unable to create output file " + outFile.getAbsolutePath(), e);
-			return false;
-		}
-		catch (final IOException e)
+		} catch (final IOException e)
```

Comment:
Curly brace standard violation

Reply:
we really need to add this to some automated tool